ovos_utils/configuration.py

```diff
@@ -42,59 +42,70 @@ def get_xdg_cache_save_path(folder=None):
     return join(xdg_cache_home(), folder)
 
 
+def _is_running_from_module(module_name):
+    #is_installed = find_spec(module_name) is not None
+    #if not is_installed:
+    #    return False
+    in_path = any([p.endswith(f"/{module_name}") for p in sys.path])
+    return in_path
+
+
 def get_ovos_config():
+    # populate default values
     config = {"xdg": True,
               "base_folder": "mycroft",
               "config_filename": "mycroft.conf"}
-
     try:
         config["default_config_path"] = find_default_config()
-    except FileNotFoundError:
-        # not a mycroft device
+    except FileNotFoundError: # not a mycroft device
         config["default_config_path"] = join(dirname(__file__), "res", "fallback_mycroft.conf")
 
-    try:
-        if isfile("/etc/OpenVoiceOS/ovos.conf"):
-            config = merge_dict(config,
-                                load_commented_json(
-                                    "/etc/OpenVoiceOS/ovos.conf"))
-        elif isfile("/etc/mycroft/ovos.conf"):
-            config = merge_dict(config,
-                                load_commented_json("/etc/mycroft/ovos.conf"))
-    except:
-        # tolerate bad json TODO proper exception (?)
-        pass
-
-    # This includes both the user config and
-    # /etc/xdg/OpenVoiceOS/ovos.conf
-    for p in get_xdg_config_dirs("OpenVoiceOS"):
-        if isfile(join(p, "ovos.conf")):
-            try:
-                xdg_cfg = load_commented_json(join(p, "ovos.conf"))
-                config = merge_dict(config, xdg_cfg)
-            except:
-                # tolerate bad json TODO proper exception (?)
-                pass
+    # load ovos.conf
+    for path in get_ovos_default_config_paths():
+        try:
+            config = merge_dict(config, load_commented_json(path))
+        except:
+            # tolerate bad json TODO proper exception (?)
+            pass
 
     # let's check for derivatives specific configs
     # the assumption is that these cores are exclusive to each other,
     # this will never find more than one override
     # TODO this works if using dedicated .venvs what about system installs?
     cores = config.get("module_overrides") or {}
     for k in cores:
-        if find_spec(k):
+        if _is_running_from_module(k):
             config = merge_dict(config, cores[k])
             break
     else:
         subcores = config.get("submodule_mappings") or {}
         for k in subcores:
-            if find_spec(k):
+            if _is_running_from_module(k):
                 config = merge_dict(config, cores[subcores[k]])
                 break
 
     return config
 
 
+def get_ovos_default_config_paths():
```

Comment:
Consider adding a method annotation with the expected paths this considers for easier debugging later